back/tests/cmd/openbsrv-integration/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/OpenEugene/openboard/back/tests/cmd/openbsrv-integration
+
+go 1.12
+
+require (
+	github.com/OpenEugene/openboard/back/internal v0.0.0-20190926014330-eca0374adab8
```

Comment:
Please drop everything after `v0.0.0` on this line, then run `go mod tidy` from within `back/tests/cmd/openbsrv-integration`.

Reply:
Added to my todo

Reply:
Done. Go mod didn't change anything for me though, which I didn't expect.